filament/src/RenderPass.cpp

```diff
@@ -579,10 +576,12 @@ RenderPass::Command* RenderPass::generateCommandsImpl(uint32_t extraFlags,
             auto const& morphTargets = morphing.targets[pi];
             FMaterialInstance const* const mi = primitive.getMaterialInstance();
             FMaterial const* const ma = mi->getMaterial();
+            bool const invertedFrontFaces =
+                    mi->isFrontFaceWindingInverted() ^ soaVisibility[i].reversedWindingOrder;
```

Comment:
`soaVisibility[i]` can be hoisted outside the loop.

Reply:
done